dockerfiles/neuroconv_latest_release_dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM python:3.11.7-slim
+LABEL org.opencontainers.image.source=https://github.com/catalystneuro/neuroconv
+RUN pip install neuroconv[full]
+CMD ["python3 -m"]
```

Comment:
I also had an error after running
```
docker run -it neuroconv_latest_release
```
```python-traceback
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "python3 -m": executable file not found in $PATH: unknown.
ERRO[0000] error waiting for container: context canceled
```
So I changed this to:
```suggestion
CMD ["python3",  "-c", "import neuroconv"]
```

Reply:
The point of a CMD is to define the prepend to the entrypoint - since we want this docker image, when run as a container, to effectively emulate an entire functioning Python environment, I'm trying to design it in such a way that it is pretty much just like calling the CLI from a normal Python environment

That is, in a typical Python NeuroConv environment you would do

```
neuroconv my_yaml_file.yml
```

now with the docker we want

```
docker run <other docker required stuff> neuroconv my_yaml_file.yml
```

While it is conceptually possible to just rely on implicit entrypoint definition from successful installation and association inside the docker container, it is much safer apparently to call `python -m` directly on the module

> CMD ["python3",  "-c", "import neuroconv"]

This simply verifies that the module is importable, it does not route the CLI - I don't even think it would facilitate pretending of a command sent to the container either

Anyway, I think the problem was `python` vs. `python3` usage. This seems to be working now, and I added a full demo if you want to check that out and see if that works for you

Reply:
Yep, seems fine